include/HUDL/HUDL.hpp

```diff
@@ -91,272 +117,92 @@ class HUDL {
     /** The torque actual value provided by the MC node over CAN. Found in the first 16 bits of the 4th PDO coming from the MC. */
     uint16_t torqueActual = 0;
 
+    /** The current position of the Motor Controller. Used to calculate speed, RPM and more. */
     int16_t actualPosition = 0;
 
+    /**
+     * A static function that retrieves the column number for a specified corner. This is used
```

Comment:
```suggestion
     * Retrieves the column number for a specified corner. This is used
```
You generally don't want to mention things about whatever you're documenting that are present in the thing itself. It just increases your chances of updating the thing and forgetting to update the comment. This includes stuff like static and any data types.

Make sure to fix this in other comments as well